### ISSUE TITLE
Trouble loading font in OS without fonts

### ISSUE THREAD

**Issue by zisra** [lovell/sharp#3479]
I'm running an app on railway and the fonts won't load (it works in macos). This is what my output should look like: 
![image](https://user-images.githubusercontent.com/100528712/205549971-f7a3a556-2f17-4a6e-97fb-7b25526fc475.png)
But instead, I'm getting this: 
![image](https://user-images.githubusercontent.com/100528712/205550085-38c786d8-905c-4456-95b0-7da20da41d8f.png)
So I thought I could try using the font and/or fontfile options (I have tried both as you can see below). All I get is `Error: Expected a valid string to create an image with text.`
```js
const data = await sharp(
				Buffer.from(SVG.map((i) => i.replace('\n', '')).join('')),
				{
					density: parseInt(density) || 300,
					text: {
						// font: 'Arial',
						fontfile: path.resolve('./src/Arial.ttf'),
					},
				}
			)
				.png({ palette: true })
				.toBuffer();
			resolve({ data, format: 'png' });
```
Arial.ttf is a valid font file, btw, and the path resolved fine for me

**Comment by zisra** [comment]
@lovell I think I misunderstood the text option. My SVG file has multiple text nodes, but unfortunately the OS does not have fonts loaded so I'm trying to use that for the existing text nodes in my SVG. Is this possible?

**Comment by lovell** [comment]
Ah, sorry, I missed the SVG bit of your question.

Please see https://sharp.pixelplumbing.com/install#fonts

> I'm running an app on railway

You'll need to work out how to configure fontconfig within the Railway runtime.

**Comment by zisra** [comment]
Closed in favor of #3485

**Comment by chaloNitin** [comment]
Which one? I'm looking for an alternative as well
